src/lib.rs

```diff
@@ -50,6 +50,7 @@ cfg_if! {
             target_os = "watchos",
             target_os = "visionos",
         )),
+        not(target_os = "vita"),
```

Comment:
Nit, this can be in the existing group

```suggestion
            target_os = "watchos",
            target_os = "vita",
        )),
```

Reply:
The reason I put it separately was due to an organization purpose: in my understanding, that group has all of the apple OSes, which is what the FIXME above applies to. Otherwise, `windows` would also be in that same group.

1. Do you still want me to make this change? 
2. Should I also do the same for windows? e.g. 
```
not(any(
    windows,
    // FIXME(ctest): just use `target_vendor` = "apple"` once `ctest` supports it
    target_os = "macos",
    target_os = "ios",
    target_os = "tvos",
    target_os = "watchos",
    target_os = "visionos",
    target_os = "vita",
)),
```

Reply:
Nope you're right, good as-is. Thanks for the clarification!